x/bundles/abci.go

```diff
@@ -46,7 +46,7 @@ func SplitInflation(ctx sdk.Context, k bundlesKeeper.Keeper, bk util.BankKeeper,
 	for _, pool := range pk.GetAllPools(ctx) {
 		// only include active pools
 		if err := k.AssertPoolCanRun(ctx, pool.Id); err == nil {
-			totalInflationShareWeight += pool.InflationShareWeight
+			totalInflationShareWeight += uint64(pool.InflationShareWeight.TruncateInt64())
```

Comment:
The reason to migrate the pool inflation share weight to a dec was so that we can have values like for example 0.2 and 0.8 instead of 2000 and 8000, please make the variable `totalInflationShareWeight` also to a dec so the calculation is accurate with every inflation share weight value